x/ccv/provider/types/errors.go

```diff
@@ -49,4 +49,5 @@ var (
 	ErrCannotCreateTopNChain                   = errorsmod.Register(ModuleName, 41, "cannot create Top N chain outside permissionlessly")
 	ErrCannotPrepareForLaunch                  = errorsmod.Register(ModuleName, 42, "cannot prepare chain for launch")
 	ErrInvalidStopTime                         = errorsmod.Register(ModuleName, 43, "invalid stop time")
+	ErrInvalidChangeRewardDenoms               = errorsmod.Register(ModuleName, 44, "invalid change reward denoms message")
```

Comment:
Is `ErrInvalidChangeRewardDenoms` used anywhere? I see in a previous [commit](https://github.com/cosmos/interchain-security/pull/2194/commits/452fe04bfc724539f34ebcb9e9b3e99b467a1487) you had:
```
// ValidateBasic is not called automatically for messages from gov proposals
if err := msg.ValidateBasic(); err != nil {
    return nil, errorsmod.Wrapf(types.ErrInvalidChangeRewardDenoms, "error: %s", err.Error())
}
```
But it was then subsequently removed. Do we need to call `ValidateBasic` for messes that stem from gov proposals or not? If we do, we should do this for `CreateConsumer` and `UpdateConsumer` as well.

Reply:
No, the gov module calls ValidateBasic for all messages when the proposal is submitted. I will remove the error.